packages/components/src/autocomplete/autocompleter-ui.js

```diff
@@ -40,7 +48,16 @@ export function getAutoCompleterUI( autocompleter ) {
 			value,
 		} );
 
+		const [ needsA11yCompat, setNeedsA11yCompat ] = useState( false );
 		const popoverRef = useRef();
+		const popoverRefs = useMergeRefs( [
+			popoverRef,
+			useRefEffect( ( node ) => {
```

Comment:
Why is there a need for `useRefEffect` ? Can't we just use a "vanilla" ref callback function?

Reply:
We're using `useRefEffect` because it's easier to understand, it works similarly to `useEffect` so you don't need to handle `null`.